src/main/groovy/de/dkfz/roddy/execution/jobs/JobManagerOptions.groovy

```diff
@@ -84,7 +84,9 @@ class JobManagerOptionsBuilder {
         requestQueueIsEnabled = true
         requestCoresIsEnabled = true
         requestStorageIsEnabled = false  // Defaults to false, not supported now.
-        passEnvironment = Optional.empty()
+        // TODO: The following is a quickfix for the LSF -env problem. Change back to the commented version, when the cluster is fixed.
+        // passEnvironment = Optional.empty()
+        passEnvironment = Optional.of(SubmissionCommand.PassEnvironmentVariables.All)   // TODO: Remove quickfix as soon as cluster is fixed.
```

Comment:
We did not talk about this, but what is the exact problem?

Reply:
The problem is that to restore the currently working behaviour in our cluster, we need to go back to copying all variables, as was done until recently. As soon as the environment is correctly created by the cluster when `bsub -env "A=B"` (i.e. without the `all`) is used, we can switch back to PassVars.Requested.